components/x-teaser/src/Video.jsx

```diff
@@ -22,7 +22,8 @@ const Embed = (props) => (
 			data-o-video-playsinline="true"
 			data-o-video-placeholder="true"
 			data-o-video-placeholder-info="[]"
-			data-o-video-placeholder-hint="Play video" />
+			data-o-video-placeholder-hint="Play video"
+			data-o-video-systemcode={props.systemCode} />
```

Comment:
this prop should be documented

Reply:
how do you document props in this library?

Reply:
oh, https://github.com/Financial-Times/x-dash/pull/422/files#diff-0ffeafa06b86f5b823888944c54c4db8R188-R192

^_^ cool